sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/newer-linker_sjs-1/project/newer-scala-js.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.0.1")

```

Comment:
IIUC we should not update the Scala.js dependency of scalajs-bundler otherwise this test won’t test anything, right?

Reply:
That's right.